From 8188843ca5c421bb113fa9492ff39b144d52a76b Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Tue, 18 Dec 2012 13:18:41 +0200
Subject: [PATCH 0384/1825] removed double SerdesPhyConfig initializations -
 added skipSerdesPhyConfig flag to indicate
 whether to run SerdesConfig ot skip it - aligned
 empty function calls with U-Boot

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 2f0cdfd392565247c03e1dc355830d743a000d08

Change-Id: Ie545d5dcd1e0ebe5954f7e6b60d0f3897e161df7

Signed-off-by: Kosta Zertsekel <konszert@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/mach-armada370/core.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-armada370/core.c b/arch/arm/mach-armada370/core.c
index df06d16..47df41c 100644
--- a/arch/arm/mach-armada370/core.c
+++ b/arch/arm/mach-armada370/core.c
@@ -1173,6 +1173,9 @@ static void __init axp_db_init(void)
 	/* init the Board environment */
 	mvBoardEnvInit();
 
+	/* skip the SerdesPhyConfig phase*/
+	mvCtrlSetSkipSerdesPhyConfig();
+
 	/* init the controller environment */
 	if( mvCtrlEnvInit() ) {
 		printk( "Controller env initialization failed.\n" );
-- 
1.7.5.4

